owl/SOMA-SAY.owl

```diff
@@ -242,10 +227,9 @@ Let xA, xP be objects filling the agent, patient roles of this schema. Then one
     
 
 
-    <!-- http://www.ease-crc.org/ont/SOMA.owl#CommandTask -->
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#CommandingTask -->
```

Comment:
As before, axioms about CommandingTask got split between soma-act and soma-say.

Reply:
See my comment above on assertiontask